onnxruntime/core/session/inference_session.h

```diff
@@ -104,8 +105,8 @@ class InferenceSession {
     for logging. This will use the default logger id in messages.
     See core/common/logging/logging.h for details, and how LoggingManager::DefaultLogger works.
     */
-  explicit InferenceSession(const SessionOptions& session_options,
-                            logging::LoggingManager* logging_manager = nullptr);
+  InferenceSession(const SessionOptions& session_options,
+                   const Environment& session_env);
```

Comment:
When we added this constructor at the start of the project there was no env class. The logging manager logically belongs to an env. This change has the potential to break the training code. Will fix it later.